src/lib/PackageVersions.js

```diff
@@ -11,15 +11,13 @@ class PackageVersions {
   }
   async resolveVersions() {
     const versions = await packageVersion(this.name);
-    this.versions = Array.from(versions).map(
-      version => new Package(this.name, version)
-    );
+    this.versions = Array.from(versions).map(version => new Package(this.name, version));
     await Promise.all(this.versions.map(pkg => pkg.resolveDependencies()));
     return true;
   }
   dependsOn(packageToUpdate: string): boolean {
     return this.versions.length
-      ? this.versions.every(pkg => pkg.dependsOn(packageToUpdate))
+      ? this.versions.find(pkg => pkg.dependsOn(packageToUpdate))
```

Comment:
`Array.prototype.find` result isn't a `boolean`(it'll return the first element that match). That's why I change this to `Array.prototype.every` 🤔 - Did I broke the app?

Reply:
to be honest, I didn't do much testing when I did #3  😛

Reply:
It did break the app because `Array.prototype.every` will test and only return true when every element has passed the test.

Reply:
At least `Array.prototype.find` will cast to true when at least one item has found

Reply:
`Array.prototype.some` might useful. Or if you insist on using `Array.prototype.find`, check the length so it return true/false.